frontend/sw360-portlet/src/main/webapp/html/projects/import.jsp

```diff
@@ -78,13 +79,23 @@
                        style="min-width:162px; min-height:28px" />
                 </div>
             </div>
-            <input type="button" onclick="updateDataSource()" value="Connect" id="buttonConnect"/>
         </div>
     </div>
     <input type="button" onclick="disconnectDataSource()" value="Disconnect" class="hidden" id="buttonDisconnect"/>
+    <div class='form-group'>
+        <label class="control-label textlabel stackedLabel" for="input-project-name">Project Name (first letters):</label>
+        <div class='controls'>
+            <input class="form-control toplabelledInput" id="input-project-name"
+                   name="<portlet:namespace/><%=Project._Fields.NAME%>"
+                   style="min-width:162px; min-height:28px" />
+        </div>
+    </div>
+    <input type="button" onclick="updateDataSource()" value="Connect" id="buttonConnect"/>
+    <input type="button" onclick="updateDataSource()" value="Refresh" class="hidden" id="buttonRefresh"/>
```

Comment:
Is there a special reason why you added a new button instead of renaming the attribute value of the first button?

Reply:
No, just that the prexisiting logic was already showing/hidding buttons instead of renaming them, so I kept using that.

Reply:
Ok